src/api/EpiData.ts

```diff
@@ -182,18 +186,24 @@ export function importCOVIDcast({
 }: {
   data_source: string;
   signal: string;
-  time_type?: 'day';
+  time_type?: string;
   geo_type: string;
   geo_value: string;
 }): Promise<DataGroup | null> {
-  const title = `[API] Delphi CODIDcast: ${geo_value} ${signal} (${data_source})`;
+  const title = `[API] Delphi COVIDcast: ${geo_value} ${signal} (${data_source}) ${time_type}`;
+  console.log(epiRange(firstEpiWeek.covidcast, currentEpiWeek));
   return loadDataSet(
     title,
     'covidcast',
-    {
-      time_type: 'day',
-      time_values: epiRange(firstDate.covidcast, currentDate),
-    },
+    time_type === 'day'
+      ? {
+          time_type: 'day',
+          time_values: epiRange(firstDate.covidcast, currentDate),
+        }
+      : {
+          time_type: 'week',
+          time_values: epiRange(firstEpiWeek.covidcast, currentEpiWeek),
+        },
```

Comment:
i dont love a big ternary like this, and we should let the `time_type` variable serve its purpose.  i would also support turning this into an `if` statement before the method call is made...
(im not 100% confident in the correctness of this suggestion)
```suggestion
    {
        time_type: time_type,
        time_values: time_type === 'day'
      ? epiRange(firstDate.covidcast, currentDate) : epiRange(firstEpiWeek.covidcast, currentEpiWeek)
    },
```